content/05.administrative.md

```diff
@@ -15,6 +15,10 @@ Finally, we would like to thank Yuanchao Zhang and Eric Wafula for adding to the
 |---|---|{% for author in manubot.authors %}
 |{{author.name}}|{% if author.contributions is string %}{{author.contributions}}{% else %}{% for contribution in author.contributions %}{{ contribution }}{% if not loop.last %}, {% endif %}{% endfor %}{% endif %}|{% endfor %}
 
+Note: Except for the first and last four authors, authorship order was determined as follows.
+Authors who contributed to the OpenPBTA code base were listed based on number of modules to which that individual contributed and randomized in the case of a tie.
+The remaining order of authors was randomized.
+Code for determining authorship order can be found in the `count-contributions` module of the OpenPBTA analysis repository.
```

Comment:
This tense makes more sense to me for talking about current order (but now how it was set)

```suggestion
Except for the first and last four authors, authorship order was determined as follows: Authors who contributed to the OpenPBTA code base are listed based on number of modules included in the manuscript to which that individual contributed and, in the case of ties, a random order is used. All remaining authors are then listed in a random order.

Code for determining authorship order can be found in the `count-contributions` module of the OpenPBTA analysis repository.
```